<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCircleCommandClass</class>
 <widget class="QDialog" name="AddCircleCommandClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddCircleCommand</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Добавить команду: Circle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Идентификатор объекта:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="objectIdInput">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Опорные точки:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="point1Input">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="point2Input">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="point3Input">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>451</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
